src/vivarium_gates_iv_iron/data/loader.py

```diff
@@ -135,6 +157,126 @@ def _load_em_from_meid(location, meid, measure):
 
 
 # TODO - add project-specific data functions here
+def load_pregnancy_incidence_rate(key: str, location: str):
+
+    not_pregnant = get_prevalence_not_pregnant(key, location)
+    asfr = load_asfr(data_keys.PREGNANCY.ASFR, location)
+    sbr = load_sbr(data_keys.PREGNANCY.SBR, location)
+    incidence_c995 = load_standard_data(data_keys.PREGNANCY.INCIDENCE_C995, location)
+    incidence_c374 = load_standard_data(data_keys.PREGNANCY.INCIDENCE_C374, location)
+
+    pregnancy_incidence_rate = (asfr + asfr * sbr + incidence_c995 + incidence_c374) / not_pregnant
+
+    return pregnancy_incidence_rate
+
+
+def load_asfr(key: str, location: str):
+
+    asfr = load_standard_data(key, location)
+
+    # pivot
+    asfr = asfr.reset_index()
+    asfr = asfr[(asfr.sex == 'Female') &
```

Comment:
Nit: I think it's more readable to put the ampersand at the start of the new line. 

Also, no need to change this, but I've recently found that the `df.query()` syntax is much cleaner and easier to use than the `[ ]` notation. Just worth investigating at some other time.